real_time/surface_src/java_src/Alice/src/test/java/TestWayPointFollowerPlanner.java

```diff
@@ -13,6 +16,8 @@
  */
 public class TestWayPointFollowerPlanner {
 
+	WayPointFollowerPlanner planner;
```

Comment:
Vasu we should have better style and git etiquette but we are still bring up the system and a large part of this commit is dependent on the other parts.  

Really we should have better code reviewing practices but this pull request came from the mad dash before race day.  

Another problem is the limited number of time/developers we have. in the 8 hours that it someone else is sleeping we tend to develop 3 or 4 new features.

hopefully next year you all will  do better